server/config/footer.json

```diff
@@ -7,6 +7,8 @@
         "facebook": "https://www.facebook.com/mathsoc",
         "twitter": "https://twitter.com/mathsoc",
         "instagram": "https://www.instagram.com/uwmathsoc/",
-        "mail": "mailto:info@mathsoc.uwaterloo.ca"
+        "mail": "mailto:info@mathsoc.uwaterloo.ca",
+        "discord": "https://mathsoc.uwaterloo.ca/how-to-access-the-mathsoc-discord/",
```

Comment:
Can this redirect to `/how-to-access-the-mathsoc-discord/` instead? (making it link to the new site instead of the wordpress one?)

Reply:
yup sure, just left this here for now since we don't have that page yet